libs/framework/include/bundle_revision.h

```diff
@@ -109,7 +101,8 @@ celix_status_t bundleRevision_getManifest(bundle_revision_pt revision, manifest_
  *      - CELIX_SUCCESS when no errors are encountered.
  *      - CELIX_ILLEGAL_ARGUMENT If <code>revision</code> is illegal.
  */
-celix_status_t bundleRevision_getHandles(bundle_revision_pt revision, celix_array_list_t **handles);
+celix_status_t bundleRevision_getHandles(const bundle_revision_t* revision, celix_array_list_t **handles)
```

Comment:
Now that the implementation is removed from bundle_revision.c, it will cause linking error for any existing application using it. I think the declaration should also be removed.

Reply:
Adding an empty implementation instead, because technically bundle_revision.h is a public header.